<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(254, 246, 221)</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>211</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(1,2,44)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnDashboard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background:rgb(1, 2, 44);*/
#btnDashboard{
font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color:#ffffff;
}
#btnDashboard:hover{
background:#fff;
border:0px;
color:rgb(1,2,44);
}</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJanitors">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>170</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background:rgb(1, 2, 44);*/
#btnJanitors{
font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color:#ffffff;
}
#btnJanitors:hover{
background:#fff;
border:0px;
color:rgb(1,2,44);
}</string>
    </property>
    <property name="text">
     <string>Janitors</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTasks">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background:rgb(1, 2, 44);*/
#btnTasks{
font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color:#ffffff;
}
#btnTasks:hover{
background:#fff;
border:0px;
color:rgb(1,2,44);
}</string>
    </property>
    <property name="text">
     <string>Tasks</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/supervisor.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background:rgb(1, 2, 44);*/
#btnLogout{
font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color:#ffffff;
}
#btnLogout:hover{
background:#fff;
border:0px;
color:rgb(1,2,44);
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>19</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Supervisor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="supervisorPages">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageDashboard">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TOTAL JANITORS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ACTIVE JANITORS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>761</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>PENDING TASKS</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>721</width>
        <height>261</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>TaskDescription</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>JanitorName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DateAssigned</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TimeAssigned</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TaskStatus</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageJanitors">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>771</width>
       <height>541</height>
      </rect>
     </property>
     <property name="title">
      <string>LIST OF JANITORS</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>751</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>JanitorName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Phone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>HireDate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EmploymentStatus</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>450</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>On Leave</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pageTasks">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>771</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>List Of Assigned Tasks</string>
     </property>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>751</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>TaskDescription</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>JanitorName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SupervisorName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DateAssigned</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TimeAssigned</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TaskStatus</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Completed</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Pending</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>771</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TaskDescription</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>JanitorName</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SupervisorName</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>19</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DateAssigned</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TimeAssigned</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TaskStatus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>140</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
